docs/develop/01_sdk/02_cookbook/06_upgrading_to_v0_29/index.md

```diff
@@ -0,0 +1,13 @@
+---
+id: howto-upgrade-v29-index
+title: Upgrading to v0.29
+---
+
+Version 0.29.0 is the result of our efforts to make the sdk easier to understand and to use.
+
+As a consequence, quite a few things have changed relative to previous versions.
+These pages serve as a reference point for what to consider when upgrading to make your transition as smooth as possible.
```

Comment:
These pages or this page?

Reply:
it's two pages plus the release notes 🤷‍♂️

Reply:
Yeah ok 😃 The main page is one though 🤣 But ok it's good in case we decide to add something else.